numpy/polynomial/_polybase.py

```diff
@@ -860,7 +860,10 @@ class domain in NumPy 1.4 and ``None`` in later versions.
         -------
         new_series : series
             A series that represents the least squares fit to the data and
-            has the domain specified in the call.
+            has the domain and window specified in the call. If the
+            coefficients for the unscaled and unshifted basis polynomials are
+            of interest, do `new_series.convert().coef
+            <numpy.polynomial.polynomial.Polynomial.convert>`
```

Comment:
I wonder if it is worth making links in the documentation? Just seems to make it harder to read in the console. So maybe just ``` ``new_series.convert().coef`` ```

Reply:
Also note that this is a base class and is used for six different polynomial classes, so the documentation needs to be generic.